entity-service/src/test/kotlin/com/egm/stellio/entity/web/EntityHandlerTests.kt

```diff
@@ -81,6 +87,8 @@ class EntityHandlerTests {
 
         every { entityService.exists(any()) } returns false
         every { entityService.createEntity(any()) } returns Entity(id = breedingServiceId, type = listOf("BeeHive"))
+        every { authorizationService.userIsCreator("mock-user") } returns true

```

Comment:
it is easier to follow if the mocked calls are in the call order (well, yes, `exists` and `createEntity` have not been swapped when code has changed)